CRM/Core/BAO/Mapping.php

```diff
@@ -649,6 +649,11 @@ public static function getBasicFields($mappingType) {
             $relationships
           );
         }
+        if (!empty($fields[$contactType]['state_province']) && empty($fields[$contactType]['state_province_name'])) {
+          $fields[$contactType]['state_province_name'] = $fields[$contactType]['state_province'];
+          $fields[$contactType]['state_province_name']['title'] = 'State Name';
+          $fields[$contactType]['state_province']['title'] = 'State Abbreviation';
```

Comment:
should'ts' be used on the titles?

Reply:
Updated @mlutfy @yashodha @colemanw 

![image](https://user-images.githubusercontent.com/5929648/147844641-f9816574-365c-4369-9b67-94b4514b5c25.png)